### PR TITLE
feat(wizard) Add notifications after applying form, add notification after save for later in form wizard

### PR DESCRIPTION
**Pull request by Radiansz**

### CODE REVIEW

**Comment by zouxuoz** [comment]
¯\_(ツ)_/¯

File: src/blocks/wizard/__step/__step.js [hunk 1]
```diff
@@ -96,6 +96,13 @@ define(function(require) {
             }).then(function() {
                 self.deactivate();
                 self.trigger('apply');
+
+                var notification = new UI.notification({
```
Comment (rosendi) [inline src/blocks/wizard/__step/__step.js]: Can we avoid using global `UI` garbage? :-)

File: src/pages/subscribers/forms/create/index.js [hunk 2]
```diff
@@ -17,9 +17,15 @@ define(function(require) {
         },
 
         redirectToEdit: function(form) {
-        	app.navigate('/subscribers/forms/edit?id=' + form.get('id'), {
-        		replace: true
-        	});
+            var notification = new UI.notification({
```
Comment (rosendi) [inline src/pages/subscribers/forms/create/index.js]: Same here with UI.

**Comment by rosendi** [comment]
LGTM

**Pull request merged**